CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # osquery Changelog
 
+
+<a name="5.2.1"></a>
+## [5.2.1](https://github.com/osquery/osquery/releases/tag/5.2.1)
+
+[Git Commits](https://github.com/osquery/osquery/compare/5.1.0...5.2.1)
+
+Osquery 5.2.1 brings native m1 support to the macOS platform. 
+
+Representing commits from 9 contributors! Thank you all.
+
+### New Features
+
+- Apple Silicon support ([#7330](https://github.com/osquery/osquery/pull/7330))
+
+### Table Changes
+
```

Comment:
I believe we dropped the `icu` dependency; we removed `lldpd` and `smartmontools` for M1 and these tables are being considered for upcoming deprecation

Reply:
Let's approach this from a user perspective. If we're going to drop `lldpd` and `smartmontools` how does that impact us? I assume the former would lose the `lldp_neighbors` table, what does the latter do?

I'd be down to put intent-to-deprecate notices somewhere in the changelog

Reply:
Ok. The `smartmontools` table has been recommended for deprecation because it was SATA-only and that limitation is not worth the maintaining of an outdated fork of the original repo. I'd tell the users that this is legacy code for legacy hardware. It remains in osquery platforms that had it, but will not go forward in M1. We can have a deprecation discussion later.

`lldp_neighbors` table ([listed on the schema page as Linux-only, but that is a bug with the schema page because it is definitely a table in macOS x86](https://osquery.io/schema/5.1.0/#lldp_neighbors)) is what would be impacted by dropping `lldpd`, but we can have that discussion later and make that announcement when ready. We can just say for now that "support for lldp_neighbors table is not in M1 at this time." On _every_ platform, it's difficult to support due to having to version-match between the library we have and the installed version of the lldpd service that it calls, and so often it may not work as intended/documented.

Right now the schema page doesn't have a way to show whether a table is available in x86-but-not-M1. That's something that will probably cause confusion.

As for `icu`, its removal does not impact anything, it was entirely unused code that was removed.

Reply:
Technical correction to my last message; it is the `smart_drive_info` table. `smartmontools` is the dependency.

Reply:
I'm thinking about _how_ to communicate this. We don't have a regular section for this in the change log. 

I'm willing to add one. I think we should create tickets for the deprecation notices and then link to them.

Reply:
Ok, the two tickets are https://github.com/osquery/osquery/issues/7463 and https://github.com/osquery/osquery/issues/7464

I suggest a section name: `Tables Being Deprecated` or `Differences in Table Support in M1` or `Known Issues with M1` linking to the label `apple_silicon` [in our issues](https://github.com/osquery/osquery/issues?q=is%3Aissue+is%3Aopen+label%3Aapple_silicon).

Reply:
Yeah, that was roughly my plan. I made a [deprecation](https://github.com/osquery/osquery/labels/deprecation) label and applied it to the 3 issues in question. And added an appropriate section for these.